docs/source/get_started/install.md

```diff
@@ -1,102 +1,192 @@
+```
+
+### How to create a new virtual environment using `conda`
+
+This is our recommended approach. From your terminal:
+
+```bash
+conda create --name kedro-environment python=3.10 -y
+```
+
+In this example, we use Python 3.10, and create a virtual environment called `kedro-environment`. You can opt for a different version of Python (any version >= 3.7 and <3.11) for your project, and name it anything you choose.
+
+The `conda` virtual environment is not dependent on your current working directory and can be activated from any directory:
 
-### Python
-Kedro supports macOS, Linux, and Windows and is built for Python 3.7+.
+```bash
+conda activate kedro-environment
+```
 
-To confirm that a valid version of Python is installed, type the following in your terminal (macOS and Linux):
+To confirm that a valid version of Python is installed in your virtual environment, type the following in your terminal (macOS and Linux):
 
 ```bash
 python3 --version
 ```
-On Windows, type the following into the command prompt:
+
+On Windows:
 
 ```bash
 python --version
 ```
 
-You should see the version of Python installed on your machine:
+To exit `kedro-environment`:
 
 ```bash
-Python 3.8.13
+conda deactivate
 ```
 
-If you see an error message or need to install a later version of Python, you can download it from the [official Python website](https://www.python.org/downloads/).
+### How to create a new Python virtual environment without using `conda`
+
+Depending on your preferred Python installation, you can create virtual environments to work with Kedro using `venv` or `pipenv` instead of `conda`.
 
-### `git`
-You will need to install `git` onto your machine if you do not already have it. To check if it is installed:
+<details>
+<summary><b>Click to expand instructions for <code>venv</code></b></summary>
+
+If you use Python 3, you should already have the `venv` module installed with the standard library. Create a directory for working with your project and navigate to it. For example:
 
 ```bash
-git -v
+mkdir kedro-environment && cd kedro-environment
 ```
 
-You should see the version of `git` available or an error message if it is not installed. You can download it from the official  [`git`](https://git-scm.com/) website.
-
-### Virtual environment manager
-We suggest you create a new Python virtual environment for *each* new Kedro project you work on to isolate its dependencies from those of other projects. We strongly recommend [installing `conda`](https://docs.conda.io/projects/conda/en/latest/user-guide/install/) if you don't already use it.
+Next, create a new virtual environment in this directory with `venv`:
 
-``` {note}
-[Read more about Python virtual environments](https://realpython.com/python-virtual-environments-a-primer/) or [watch an explainer video about them](https://youtu.be/YKfAwIItO7M).
+```bash
+python -m venv env/kedro-environment  # macOS / Linux
+python -m venv env\kedro-environment  # Windows
```

Comment:
Let's avoid the extra nesting here:

````suggestion
```bash
python -m venv .venv
```
````

(and also this suggestion is cross platform)